openshift-client/src/test/java/io/fabric8/openshift/client/mock/RoleBindingTest.java

```diff
@@ -0,0 +1,249 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.openshift.client.mock;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.fabric8.openshift.api.model.RoleBinding;
+import io.fabric8.openshift.api.model.RoleBindingBuilder;
+import io.fabric8.openshift.client.NamespacedOpenShiftClient;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RoleBindingTest {
+  @Rule
+  public OpenShiftServer server = new OpenShiftServer();
+
+  private RoleBinding expectedRoleBinding = new RoleBindingBuilder()
+    .withNewMetadata().endMetadata()
+    .addToUserNames("testuser1", "testuser2", "system:serviceaccount:testns:svcacct")
+    .addToGroupNames("testgroup")
+    .addNewSubject().withKind("User").withName("testuser1").endSubject()
+    .addNewSubject().withKind("User").withName("testuser2").endSubject()
+    .addNewSubject().withKind("ServiceAccount").withNamespace("testns").withName("svcacct").endSubject()
+    .addNewSubject().withKind("Group").withName("testgroup").endSubject()
+    .build();
```

Comment:
Does that sound OK @rhuss? Would like to merge ASAP & get a release out if possible.